jwst/outlier_detection/outlier_detection.py

```diff
@@ -165,20 +165,13 @@ def do_detection(self):
 
         # Perform median combination on set of drizzled mosaics
         median_model.data = self.create_median(drizzled_models)
-        if pars['save_intermediate_results']:
-            if self.outlierpars.get('asn_id', None) is None:
-                suffix_to_remove = self.resample_suffix
-            else:
-                suffix_to_remove = f"_{self.outlierpars['asn_id']}{self.resample_suffix}"
-            median_model_output_path = self.make_output_path(
-                basepath=median_model.meta.filename.replace(suffix_to_remove, '.fits'),
-                suffix='median')
-            median_model.save(median_model_output_path)
-            log.info(f"Saved model in {median_model_output_path}")
+
+        if self.outlierpars['save_intermediate_results']:
```

Comment:
Why is all the complex file name handling in (old) lines 168-177 being removed? I see to recall folks spending a lot of time getting all of that to work just right.

Reply:
Oh wait, it looks like all of that stuff has just been moved to the new `save_median()` method, right?

Reply:
yes, that's correct